games/static/games/proto.js

```diff
@@ -7,22 +7,40 @@ class Game {
     this.restartButton = document.querySelector("#restart");
     this.isRunning = false
   }
+  win(board) {
```

Comment:
doesnt game contains the board as its class member? if so, you dont really need to pass it as a parameter.
about the name - the name of the method doesnt indicates that it returns a boolean value
i would expect something like hasDefenderWon() that its ovious it returns true or false

Reply:
board is created in game.start() it is not a class member. if you think it should be i could make it.

Reply:
anyway i changed the name to allAliensDeadOn(board) and i took it out of Game so now the line reads:
if (allAliensDeadOn(board)){}